README.md

```diff
@@ -1,17 +1,19 @@
 # BT-Speaker
 
 A simple Bluetooth Speaker Daemon designed for the Raspberry Pi 3.
+This is a fork of (https://github.com/lukasjapan/bt-speaker/blob/master/install.sh).
+
 
 ## Installation
 
 Quick Installation for Raspbian:
 
 ```bash
 sudo -i
-bash <(curl -s https://raw.githubusercontent.com/lukasjapan/bt-speaker/master/install.sh)
+bash <(curl -s https://raw.githubusercontent.com/nuthub/bt-speaker/master/install.sh)
 ```
 
-For details refer to the comments in the [install script](https://github.com/lukasjapan/bt-speaker/blob/master/install.sh).
+For details refer to the comments in the [install script](https://github.com/nuthub/bt-speaker/blob/master/install.sh).
```

Comment:
Same here...

Reply:
I like the idea with the pin and espeak!
And also pulseaudio sounds better, because it's nice sound quality on a Raspberry

Reply:
I think these changes got in here because created pull request from their master branch.